CHANGELOG-npm.md

```diff
@@ -6,6 +6,7 @@
 - btc: handle error when an input's previous transaction is required but missing
 - btc: add support for regtest
 - btc: add support for Taproot wallet policies
+- eth: add method to help clients identify address case (upper/lower/mixed)
```

Comment:
it's not just that, it also allows clients to specify the address case in the first place when signing a tx.